ooni/errors.py

```diff
@@ -148,6 +149,9 @@ def failureToString(failure):
     elif isinstance(failure.value, ProcessDone):
         string = 'process_done'
 
+    elif isinstance(failure.value, ConnectionDone):
+        string = 'connection_done'
```

Comment:
BTW, @srvetus refactored this file in https://github.com/TheTorProject/ooni-probe/pull/413. If that PR is merged i should update this